en/discover/deployment/articles/06-upgrading-to-liferay-7-1/02-preparing-for-upgrade-to-liferay-7-1/00-preparing-for-upgrade-to-liferay-7-1-intro.markdown

```diff
@@ -118,49 +118,54 @@ provide property details and examples.
 
 ## Step 6: Configuring Your Documents and Media File Store
 
-It's time to migrate and update your document store configuration to
-@product-ver@. Here's what's changed for document stores:
+It's time to migrate and update your document store to
+@product-ver@.
 
--   Store implementation class package names changed from 
+1.  Store implementation class package names changed from 
     `com.liferay.portlet.documentlibrary.store.*` in Liferay Portal 6.2 to
     `com.liferay.portal.store.*` in @product@ 7.0+. Make sure your
     `portal-ext.properties` sets the `dl.store.impl` property one of these ways:
 
-        dl.store.impl=com.liferay.portal.store.file.system.FileSystemStore
-        dl.store.impl=com.liferay.portal.store.db.DBStore
-        dl.store.impl=com.liferay.portal.store.file.system.AdvancedFileSystemStore
-        dl.store.impl=com.liferay.portal.store.s3.S3Store
+    ```properties
+    dl.store.impl=com.liferay.portal.store.file.system.FileSystemStore
+    dl.store.impl=com.liferay.portal.store.db.DBStore
+    dl.store.impl=com.liferay.portal.store.file.system.AdvancedFileSystemStore
+    dl.store.impl=com.liferay.portal.store.s3.S3Store
+    ```
 
--   JCR Store was deprecated as of Liferay DXP Digital Enterprise 7.0 Fix Pack 
+1.  JCR Store was deprecated as of Liferay DXP Digital Enterprise 7.0 Fix Pack 
     14 and Liferay Portal CE 7.0 GA4. The
     [Document Repository Configuration](/docs/7-1/deploy/-/knowledge_base/d/document-repository-configuration)
     documentation describes other store options.
     [Migrate your document data](/docs/7-1/user/-/knowledge_base/u/server-administration)
     to one of the other store options before upgrading from your current Liferay
     version. 
 
--   Since @product@ 7.0, document store specific configuration (e.g., 
+1. If you're using Advanced File System Store or Simple File System Store, copy the document ling Advanced File System Store or Simple File System Store, copy the document library's root folder to your new installation or make sure the the newibrary's root folder to your new installation or make sure the the new installation can access it.
```

Comment:
What is a "document ling?" ;-)

Reply:
Document lings are all the rage. :-D

Yes, a copy/paste error in both branches. I'll fix. Thanks!

Reply:
OK. It's fixed now and ready for review. Thanks!